tests/test_add_start_node.py

```diff
@@ -0,0 +1,38 @@
+from selenium.webdriver.remote.webdriver import WebDriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver import ActionChains
+from selenium.common.exceptions import NoSuchElementException
+
+web = "https://machinist.flapflap.io/"
+
+
+def test_add_node(driver: WebDriver):
+
+    driver.get(web)
+    source = driver.find_element(By.XPATH, '//*[@id="__next"]/main/div[1]')
+    action = ActionChains(driver)
+    action.double_click(source).perform()
+
+    # check if node is added by verifying if there is text q1
+    inspect_val = driver.find_element(
+        By.XPATH, '//*[@id="__next"]/main/div[1]/div/div[2]/p'
+    ).text
+    true_val = "q1"
```

Comment:
Note: since it is the first node that you are creating that node should be q0 and q1 the reason why this test is passing right now is that there is a bug FLAP-82 which generate the q0 node automatically but once the bug is is fixed this test will fail

what I recommend is still set true_val ='q0' and comment the assertion until the bug is fixed